Jenkinsfile

```diff
@@ -102,6 +102,7 @@ pipeline {
                         sh 'cd build && make check-clang-tidy'
                         sh 'cd build && gtimeout 1h make unittest'
```

Comment:
I think you need to kill the server before the `make unittest'` target too.

Reply:
done